src/main/preprocessing/intermediate_changes_removing.py

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) 2020 Anastasiia Birillo, Elena Lyulina
+
+import logging
+import pandas as pd
+from typing import Optional
+
+from src.main.util import consts
+from src.main.util.consts import LOGGER_NAME, TMP_COLUMN
+from src.main.util.data_util import handle_folder
+
+log = logging.getLogger(LOGGER_NAME)
+
+FRAGMENT = consts.CODE_TRACKER_COLUMN.FRAGMENT.value
+SHIFT_FRAGMENT = TMP_COLUMN.SHIFT_FRAGMENT.value
+CHANGES = TMP_COLUMN.CHANGES.value
+SHIFT_CHANGES = TMP_COLUMN.SHIFT_CHANGES.value
+
+
+def __line_number_with_same_fragments(current_fragment: str, next_fragment: str) -> Optional[int]:
```

Comment:
get_line_number_with_diffs? get_diffs_line_number? 
Also, please, write a small description of this function, describing that this function returns a line number with diffs if there were diffs in only one line, and None otherwise